### DISCUSSION TITLE
[Feature] Be able to hide pictures who has been added in album from main library

### DISCUSSION THREAD

**Discussion by LespiletteMaxime** [immich-app/immich#4247] [Feature Request]
### The feature

Hello,
Thanks for immich it is a great software

My request feature is the possibility to hide from library the picture who has been added in album, can help to sort pictures. Or maybe add an Orphan tab who'll gives the picture who are not in an album already.

Thanks.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2838
